mappings/net/minecraft/world/gen/surfacebuilder/SurfaceRules.mapping

```diff
@@ -105,16 +119,21 @@ CLASS net/minecraft/unmapped/C_flergppe net/minecraft/world/gen/surfacebuilder/S
 			ARG 5 biomeRegistry
 			ARG 6 heightContext
 		METHOD m_bgmaggvu getMinSurfaceLevel ()I
+		METHOD m_dgelrfgw getOrComputeSurfaceSecondary ()D
 		METHOD m_ibujkmil updateHorizontalContext (II)V
 			ARG 1 x
 			ARG 2 z
+		METHOD m_ntlmoytt blockCoordToSurfaceCell (I)I
```

Comment:
Can't think of any better name

Reply:
honestly me neither, id just keep it. It isn't that big of a deal, it just was a bit confusing to see both of them together. Logically, it makes sense though since both methods I presume are polar opposites.

Reply:
if anyone else can give input that would be great